tests/everest/test_everest_output.py

```diff
@@ -20,14 +20,9 @@
 from everest.util import makedirs_if_needed
 
 
-def test_that_one_experiment_creates_one_ensemble_per_batch(
-    copy_math_func_test_data_to_tmp, evaluator_server_config_generator
-):
-    config = EverestConfig.load_file("config_minimal.yml")
-
-    run_model = EverestRunModel.create(config)
-    evaluator_server_config = evaluator_server_config_generator(run_model)
-    run_model.run_experiment(evaluator_server_config)
+def test_that_one_experiment_creates_one_ensemble_per_batch(cached_example):
```

Comment:
Maybe this could be sped up a bit more by limiting the number of batch, e.g. to two?

Reply:
Oh, guess not, its not running the case anyway?